app_package/src/vaporwearExperience.ts

```diff
@@ -0,0 +1,124 @@
+import "@babylonjs/loaders";
+import "@babylonjs/gui";
+
+import { Engine, SceneLoader } from "@babylonjs/core";
+import { Showroom, ShowroomState } from "./showroom";
+import { IVaporwearExperienceParams } from "./iVaporwearExperienceParams";
+import { GLTFFileLoader } from "@babylonjs/loaders";
+
+export interface IHotspotState {
+    x: number;
+    y: number;
+    isVisible: boolean;
+}
+
+export class VaporwearExperience {
+    private _showroom: Showroom;
+
+    private constructor(showroom: Showroom) {
+        this._showroom = showroom;
+    }
+
+    public static async CreateAsync(params: IVaporwearExperienceParams): Promise<VaporwearExperience> {
+        SceneLoader.OnPluginActivatedObservable.add((plugin) => {
+            if (plugin.name === "gltf") {
+                const loader = plugin as GLTFFileLoader;
+                loader.transparencyAsCoverage = true;
+            }
+        });
+
+        const canvas = params.canvas;
+        const engine = new Engine(canvas, undefined, {
+            forceSRGBBufferSupportState: true
+        });
```

Comment:
Maybe useful to link to documentation for this since it's kind of random.
https://doc.babylonjs.com/divingDeeper/3D_commerce_certif#certified-viewer-version-based-on-babylonjs-engine